container-images/sumo/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM debian:10 AS build
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
+	apt-get install -y --no-install-recommends \
+	cmake \
+	g++ \
+	git \
+	ca-certificates\
+	libfox-1.6-dev \
+	libgdal-dev \
+	libgl2ps-dev swig \
+	libproj-dev \
+	libxerces-c-dev \
+	python3-setuptools \
+	python3-dev \
+	&& \
+	apt-get autoremove -y && \
+	apt-get clean -y && \
+	rm -rf /var/lib/apt/lists/*
+
+ENV DEBIAN_FRONTEND=false
+
+ARG SUMO_TAG
```

Comment:
Can we use default values for ARGs in Dockerfiles? It would be nice if `docker build .` would just work...

Reply:
Ok, then we remove the versioning logic from the Makefile.